datahub-web-react/src/utils/customGraphQL/useGetAllEntitySearchResults.ts

```diff
@@ -0,0 +1,44 @@
+import { EntityType } from '../../types.generated';
+import { useGetSearchResultsQuery } from '../../graphql/search.generated';
+
+export function useGetAllEntitySearchResults(input: any) {
```

Comment:
we should be able to get this type from search.generated- you don't need to leave this as an any

Reply:
Specifically it should be `SearchInput` from `src/types.generated.ts`

Reply:
If you want to allow any subset of fields from the type you could consider using `Partial<SearchInput>` as the type

Reply:
Done!

Reply:
Thanks for this information about `Partial`